pkgs/by-name/be/better-control/package.nix

```
@@ -0,0 +1,103 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  gtk3,
+  dbus,
+  networkmanager,
+  bluez,
+  pipewire,
+  brightnessctl,
+  power-profiles-daemon,
+  gammastep,
+  libpulseaudio,
+  desktop-file-utils,
+  wrapGAppsHook4,
+  gobject-introspection,
+  usbguard,
+  upower,
+  nix-update-script,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "better-control";
+  version = "v6.11.6";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "quantumvoid0";
+    repo = "better-control";
+    tag = version;
+    hash = "sha256-+2hY+o+GPyJHXpQFVW8BOUEiIBGQ1hItOVpA/AVas2Q=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    wrapGAppsHook4
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    gtk3
+    dbus
+    libpulseaudio
+    networkmanager
+    bluez
+    pipewire
+    brightnessctl
+    power-profiles-daemon
+    gammastep
+    usbguard
+    upower
+  ];
```

```suggestion
  ];
  
  # Check src/utils/dependencies.py
  runtimeDeps = [
    libpulseaudio
    networkmanager
    bluez
    brightnessctl
    power-profiles-daemon
    gammastep
    usbguard # not listed, in src/ui/tabs/usbguard_tab.py
    upower
  ];
```

dbus is called by `pydbus`. pipewire is unused (it uses `pactl`, which goes through pulse compatibility layer of pipewire).

---

Please preserve my comment for further reference.

---

Which comment this

` # Check src/utils/dependencies.py`

or should I add this comment

`dbus is called by pydbus. pipewire is unused (it uses pactl, which goes through pulse compatibility layer of pipewire).`